Rules/mtbmap.mrules

```diff
@@ -1880,6 +1891,9 @@ rules
 		draw : fill
 
 
+
+// --
+		
```

Comment:
Here and in several other places there are multiple consecutive blank lines. 
I'm not sure they add to the readability of the file.